Games/types/Mafia/roles/cards/CureAllMadness.js

```diff
@@ -5,32 +5,16 @@ module.exports = class CureAllMadness extends Card {
   constructor(role) {
     super(role);
 
-    this.meetings = {
-      "Cure Madness": {
-        states: ["Night"],
-        flags: ["voting"],
-        action: {
+    this.actions = [
+      {
           priority: PRIORITY_EFFECT_GIVER_DEFAULT + 1,
```

Comment:
effect taker default?

Reply:
there is no Priority_effect_taker_default, this card always used effect_giver. Would you want me to create a new priority? can do